.gitignore

```diff
@@ -15,3 +15,5 @@
 /.release
 .elixir.plt
 erl_crash.dump
+/lib/*/cover
+/lib/*/_build
```

Comment:
That isn't correct way to run tests for logger, we need to use `mix` from the repository rather than globally installed (with homebrew for example).
There is a `make test_logger` command to do the right thing.